README.md

```diff
@@ -5,6 +5,14 @@
 
 This is a Java library which generates Lombok-wired JPA entity source code. The project provides Gradle plugin and Maven plugin.
 
+### update
```

Comment:
I know we currently don't have CHANGELOG file in this project but we can have these on CHANGELOG we can newly create, not on README.